File: Special Day.py
x = 2
y = 18

a = int(input("Enter a month (1-12): "))
b = int(input("Enter a day (1-31): "))

if a == x:
    if b == y:
        print("SPECIAL")
    elif b > y:
        print("After")
    else:
        print("Before")
elif a < x:
    print("Before")
else:
    print("After")

File: CPP/2015/SpecialDay.cpp
// J1

#include <iostream>

using namespace std;

int main() {
    const int specialmonth = 2, specialday = 18;
    int month, day;

    cin >> month;
    cin >> day;

    if (month < specialmonth) cout << "Before" << endl;
    else if (month > specialmonth) cout << "After" << endl;
    else {
        if (day < specialday) cout << "Before" << endl;
        else if (day > specialday) cout << "After" << endl;
        else cout << "Special" << endl;
    }
}
